rllib/agents/a3c/a2c.py

```diff
@@ -14,12 +15,31 @@
         "sample_batch_size": 20,
         "min_iter_time_s": 10,
         "sample_async": False,
+
+        # A2C supports microbatching, in which we accumulate gradients over
+        # batch of this size until the train batch size is reached. This allows
+        # training with batch sizes much larger than can fit in GPU memory.
+        # To enable, set this to a value less than the train batch size.
+        "microbatch_size": None,
     },
 )
 
+
+def choose_policy_optimizer(workers, config):
+    if config["microbatch_size"]:
```

Comment:
should throw an error if micro_batch > train batch size

Reply:
Yep that's done already in the optimizer.